webapp/components/posts_view_container.jsx

```diff
@@ -60,17 +59,14 @@ export default class PostsViewContainer extends React.Component {
         ChannelStore.addLeaveListener(this.onChannelLeave);
         PostStore.addChangeListener(this.onPostsChange);
         PostStore.addPostsViewJumpListener(this.handlePostsViewJumpRequest);
-        UserStore.addChangeListener(this.onUserChange);
+        $('body').addClass('app__body');
```

Comment:
It was being done inside post_views.jsx which was causing it to be applied and unapplied on each mount/unmount of one of those. I just put it here so it's added once and removed once.